C7GameData/RulesFormat.cs

```diff
@@ -43,6 +43,9 @@ public class C7RulesFormat
 		public StrengthBonus cityLevel2DefenseBonus;
 		public StrengthBonus cityLevel3DefenseBonus;
 
+		public int healRateInField;
+		public int healRateInCity;
```

Comment:
really, there should be options for friendly, neutral, and hostile field, possibly in transport, and I suppose any other conditions (eg fortress) would be modifiers

Reply:
Do fortresses affect heal rate in the original game?

Reply:
They don't.  I agree that _long-term_, we should allow more flexibility.  However, this is "Combat Part II", so my opinion is we should write up what WildWeazel wrote as an "enhancement" issue, but not try to add it in here.  A big question will be which enhancements such as this do we prioritize, but they aren't all necessary for compatibility.